packages/block-library/src/image/index.php

```diff
@@ -234,6 +229,7 @@ class="lightbox-trigger"
 			type="button"
 			aria-haspopup="dialog"
 			aria-label="' . esc_attr( $aria_label ) . '"
+			aria-describedby="' . esc_attr( $alt ) . '"
```

Comment:
The `aria-describedby` attribute should point to the ID of a hidden element that contains the description. If it turns to be too complicated, we could consider to not use a description at all.